### PR TITLE
Np 125 fe payment page payment summary

### PR DESCRIPTION
**Pull request by maxkanin**
## **Ticket**

**JIRA ID:** [NP-125](https://parking-system.atlassian.net/browse/NP-125)

## Changes proposed in this pull request
- [IMPORTANT] Reviewer **will not review** any PR without summary of changes
- add pay the bill UI 

please fill all the required fields from below Checklist otherwise reviewer **will not review** this PR
## Checklist
- [x] Screenshot evidence for both IOS and Android
- [x] Total change is less than 500 lines of code

## **Screenshot Evidence**

- Android

https://github.com/ParkingSystemWithLPR/Napark-application/assets/88888222/0a2fb463-0314-4326-a21d-d24ff42bdfab

- IOS

https://github.com/ParkingSystemWithLPR/Napark-application/assets/88888222/d0bfeb31-6832-4bf3-9e8a-79a78b8c9b04

[NP-125]: https://parking-system.atlassian.net/browse/NP-125?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by maxkanin** [comment]
Android
image

**Comment by maxkanin** [comment]
IOS
image

**Comment by Petoxic** [comment]
Have you make the case when credit is not enough yet? Not seen in a video.

**Comment by maxkanin** [comment]
เดี๋ยวจัดการตอนกดปุ่ม pay the bill ในภายหลังแบบ ถ้า เลือก QR navigate ไป หน้า QR ถ้า เลือก credit ถ้าไม่พอ ไปหน้าtop up อะไรประมาณนี้

**Comment by maxkanin** [comment]
image

**Review by Petoxic — APPROVED** [review]

File: screens/payment/PayTheBill.tsx [hunk 1]
```diff
@@ -1,13 +1,108 @@
...
+    backgroundColor: Colors.white,
+    shadowColor: Colors.black,
+    borderRadius: 8,
+    shadowOffset: {
+      width: 0,
+      height: 2,
+    },
+    shadowOpacity: 0.2,
+    shadowRadius: 2,
+    elevation: Platform.OS === "android" ? 4 : 2,
```
Comment (TaeCV) [inline screens/payment/PayTheBill.tsx]: you can remove checking Platform logic since IOS doesn't support this attribute

**Review by TaeCV — APPROVED** [review]

**Pull request merged**